workshops/models.py

```diff
@@ -481,7 +481,7 @@ class Event(models.Model):
     REPO_FORMAT = 'https://github.com/{name}/{repo}'
     WEBSITE_REGEX = re.compile(r'https?://(?P<name>[^.]+)\.github\.'
                                r'(io|com)/(?P<repo>[^/]+)/?')
-    WEBSITE_FORMAT = 'https://{name}.github.io/{repo}'
+    WEBSITE_FORMAT = 'https://{name}.github.io/{repo}/'
```

Comment:
It's just a formatting. So it works like this: 
1. try to match known regex against some URL
2. Success? Format it with `WEBSITE_FORMAT` or `REPO_FORMAT` (depending on whether it's `Event.website_url` or `Event.repository_url`)
3. Failure? Just return the original URL

This way, since most of the URLs are from GitHub, we can match & transform most of them, but for the links we cannot match we still have the original URL.